ledger-api/grpc-definitions/com/daml/ledger/api/v1/experimental_features.proto

```diff
@@ -17,7 +17,38 @@ option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 // See the feature message definitions for descriptions.
 message ExperimentalFeatures {
   ExperimentalSelfServiceErrorCodes self_service_error_codes = 1;
+  CommandDeduplicationFeatures command_deduplication = 2;
 }
 
 // GRPC self-service error codes are returned by the Ledger API.
 message ExperimentalSelfServiceErrorCodes {}
+
+
+message CommandDeduplicationFeatures {
+  DeduplicationPeriodSupport deduplication_period_support = 1;
+  ParticipantDeduplicationSupport participant_deduplication_support = 2;
+}
+
+message DeduplicationPeriodSupport {
+  enum OffsetSupport {
+    // Ledgers use offsets for command deduplication
+    OFFSET_NATIVE_SUPPORT = 0;
+    // Ledger convert offsets into a duration for command deduplication
+    OFFSET_CONVERT_TO_DURATION = 1;
+    // Ledger does not support offsets for command deduplication
+    OFFSET_NOT_SUPPORTED = 2;
+  }
+  enum DurationSupport {
+    DURATION_NATIVE_SUPPORT = 0;
+    DURATION_CONVERT_TO_OFFSET = 1;
+  }
+  OffsetSupport offset_support = 1;
+  DurationSupport duration_support = 2;
+}
+
+// Ledger support for participant deduplication
+enum ParticipantDeduplicationSupport {
+  PARTICIPANT_DEDUPLICATION_NOT_SUPPORTED = 0;
+  PARTICIPANT_DEDUPLICATION_PARALLEL_ONLY = 1;
+  PARTICIPANT_DEDUPLICATION_SUPPORTED = 2;
```

Comment:
What do these values mean? Add comments :)

Reply:
Thank you @fabiotudone-da for adding the descriptions